scripts/npm-publish.sh

```
@@ -12,5 +12,6 @@ if npm view "${PKG_NAME}@${PKG_VER}" version > /dev/null 2>&1; then
   echo "already published: ${PKG_NAME}@${PKG_VER} — skipping"
 else
   bun pm pack
-  npm publish ./$PKG_NAME-$PKG_VER.tgz --provenance --access public $flags
+  cd $project_dir
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)